gulp-tasks/test:build-bundles.js

```diff
@@ -0,0 +1,101 @@
+const path = require('path-extra');
+const gulp = require('gulp');
+const fs = require('fs-extra');
+const rollup = require('rollup-stream');
+const source = require('vinyl-source-stream');
+const resolve = require('rollup-plugin-node-resolve');
+const commonjs = require('rollup-plugin-commonjs');
+const replace = require('rollup-plugin-replace');
+const multiEntry = require('rollup-plugin-multi-entry');
+
+const constants = require('./utils/constants');
+const packageRunnner = require('./utils/package-runner');
+const logHelper = require('./utils/log-helper');
+const pkgPathToName = require('./utils/pkg-path-to-name');
+
+const buildTestBundle = (packagePath, runningEnv, nodeEnv) => {
+  const testPath = path.join(packagePath, 'test');
+  const environmentPath = path.join(testPath, 'bundle', runningEnv);
+
+  // First check if the bundle directory exists, if it doesn't
+  // there is nothing to build (NOTE: Rollup + multientry will
+  // always create a file, even if the directory doesn't exist)
+  if (!fs.pathExistsSync(environmentPath)) {
+    return Promise.resolve();
+  }
+
+  logHelper.log(`Building Test Bundle for ` +
```

Comment:
I know I've asked about this in the past (and been shot down), but can we take this opportunity to either pull in https://github.com/declandewet/common-tags#oneline or implement similar logic in our own internal helper library?

We're using template strings, so we might as well take advantage full of them, and be able to write this as

```js
logHelper.log(oneLine`
  Building Test Bundle for
  ${logHelper.highlight(pkgPathToName(packagePath))}
  to run in '${logHelper.highlight(runningEnv)}'
  with NODE_ENV='${logHelper.highlight(nodeEnv)}'.
`);
```

(Your function call also uses a combination of both `+` and `,`, inadvertently mixing concatenation along with multiple string parameters, which emphasizes the fact that it's easy to be inconsistent with the current approach.)